APIcall/data_collection.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import argparse
+from datetime import date
+
+def api_311(start_date, end_date, skip, limit):
+    skip = 0
+    limit = 10000
+    data_year = []
+    while True:
+        url=f'https://dev-api.311-data.org/requests?start_date={start_date}&end_date={end_date}&skip={skip}&limit={limit}'
+        response = requests.get(url)
+        data = response.json()
+        if data == []:

```

Comment:
Since we do not have information about the number of pages here, I am breaking the loop if there are no more pages left to fetch, i.e., when data is empty.